tests/gatsby-starter-default/src/components/Breadcrumbs.js

```diff
@@ -9,7 +9,9 @@ export default ({ data }) => (
       </li>
       {data._components.breadcrumbs.items.map(item =>
         <li key={item._id} className={'breadcrumb-item'}>
-          {item.title}
+          <Link to={`/${item._path}`}>
```

Comment:
@ajayns Shouldn’t we make such _paths that we would not need to add extra slashes in links?

Reply:
Yes, true.